File: app/src/main/java/net/ginteam/carmen/model/Point.java
package net.ginteam.carmen.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by vadik on 23.01.17.
 */

public class Point implements Serializable {

    @SerializedName("lat")
    private double mLatitude;

    @SerializedName("lng")
    private double mLongitude;

    public double getLatitude() {
        return mLatitude;
    }

    public double getLongitude() {
        return mLongitude;
    }
}

File: app/src/main/kotlin/net/ginteam/carmen/kotlin/model/Point.kt
package net.ginteam.carmen.kotlin.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 * Created by eugene_shcherbinock on 2/9/17.
 */

data class PointModel(@SerializedName("lat") val latitude: Double,
                      @SerializedName("lng") val longitude: Double) : Serializable